controllers/component_dependency_update_controller.go

```diff
@@ -424,7 +437,7 @@ func DefaultDependenciesUpdate(ctx context.Context, client client.Client, scheme
 		return false, err
 	}
 	log.Info("creating renovate job")
-	err = CreateRenovaterJob(ctx, client, scheme, installationsToUpdate, slug, true, generateRenovateConfigForNudge, result)
+	err = CreateRenovaterPipeline(ctx, client, scheme, result.Component.Namespace, installationsToUpdate, slug, true, generateRenovateConfigForNudge, result)
```

Comment:
Why pipeline and not a job?

Reply:
- Job's are removed after a timeout, so lots of nudges in a short period could make the namespace unusable due to pod Quota, while pipelines are pruned much more aggressively
- Users can view the results of pipelines and the results are stored, making debugging much easier
- Tekton automatically provides docker config from linked service accounts for private images, with a job I would need to implement this manually

Reply:
I am not sure if we should change the other job to also use a pipeline, but as we don't really have e2e tests I don't really want to make major changes to that code just before I go on PTO.

Reply:
I see, thanks